WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -5,6 +5,11 @@ class EditCommentTableViewController: UITableViewController {
 
     // MARK: - Properties
 
+    private var authorName: String?
+    private var commentContent: String?
+    private var authorWebAddress: String?
+    private var authorEmailAddress: String?
```

Comment:
Comment data is stored separately here on purpose (as opposed to just storing the Comment). Notifications doesn't use a `Comment` object, so it will have to provide the data separately. Having each stored separately here will accommodate for that.